dev/TabView/TabView.cpp

```diff
@@ -86,19 +89,96 @@ void TabView::OnApplyTemplate()
         return addButton;
     }());
 
+    // KeyboardAccelerator is only available on RS3+
     if (SharedHelpers::IsRS3OrHigher())
     {
-        winrt::KeyboardAccelerator keyboardAccelerator;
-        keyboardAccelerator.Key(winrt::VirtualKey::F4);
-        keyboardAccelerator.Modifiers(winrt::VirtualKeyModifiers::Control);
-        keyboardAccelerator.Invoked({ this, &TabView::OnCtrlF4Invoked });
-        keyboardAccelerator.ScopeOwner(*this);
-        KeyboardAccelerators().Append(keyboardAccelerator);
+        winrt::KeyboardAccelerator ctrlf4Accel;
+        ctrlf4Accel.Key(winrt::VirtualKey::F4);
+        ctrlf4Accel.Modifiers(winrt::VirtualKeyModifiers::Control);
+        ctrlf4Accel.Invoked({ this, &TabView::OnCtrlF4Invoked });
+        ctrlf4Accel.ScopeOwner(*this);
+        KeyboardAccelerators().Append(ctrlf4Accel);
```

Comment:
If the template is re-applied does this mean we add more to KeyboardAccelerators() ? Maybe it doesn't matter?

Reply:
Good point. There is no need for logic to even be in OnApplyTemplate. Moved to ctor.